tools/TrimDeps/Program.cs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using System.Linq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        ChangeEntryPointLibraryName(args[0]);
+    }
+
+    private static void ChangeEntryPointLibraryName(string depsFile)
+    {
+        JToken deps;
+        using (var file = File.OpenText(depsFile))
+        using (JsonTextReader reader = new JsonTextReader(file))
+        {
+            deps = JObject.ReadFrom(reader);
+        }
+
+        string version = null;
+        foreach (JProperty target in deps["targets"])
+        {
+            var targetLibrary = target.Value.Children<JProperty>().FirstOrDefault();
```

Comment:
Is this logic too fragile? Are we guaranteed that the first entry is always the one we want to remove? Should we not check the name of the assembly or something?